### ISSUE TITLE
error cannot read properties of null (reading 'scrollTop')

### ISSUE THREAD

**Issue by ahmedrowaihi** [sanity-io/sanity#6949]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `sanity@3.45.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/sanity/lib/index.mjs b/node_modules/sanity/lib/index.mjs
index c8fb18c..d29aff6 100644
--- a/node_modules/sanity/lib/index.mjs
+++ b/node_modules/sanity/lib/index.mjs
@@ -795,7 +795,7 @@ const getOffsetsTo$1 = (source, target) => {
   };
   let foundScrollContainer = !1, el = source;
   for (; el && el !== target && target.contains(el); )
-    foundScrollContainer && (bounds.top += el.offsetTop, bounds.left += el.offsetLeft), hasOverflowScroll(el) && (bounds.top = el.offsetTop, bounds.height = el.offsetHeight, bounds.left = el.offsetLeft, bounds.width = el.offsetWidth, foundScrollContainer = !0), rect.top += el.offsetTop - el.scrollTop, rect.left += el.offsetLeft - el.scrollLeft, el = el.offsetParent;
+    foundScrollContainer && (bounds.top += el.offsetTop, bounds.left += el.offsetLeft), hasOverflowScroll(el) && (bounds.top = el.offsetTop, bounds.height = el.offsetHeight, bounds.left = el.offsetLeft, bounds.width = el.offsetWidth, foundScrollContainer = !0), rect.top += el.offsetTop - (el?.scrollTop || 0), rect.left += el.offsetLeft - el.scrollLeft, el = el.offsetParent;
   return { rect, bounds };
 };
 function isChangeBar(v) {
@@ -8408,6 +8408,7 @@ function group(regionsWithIntersectionDetails) {
   };
 }
 const Spacer = ({ height, ...rest }) => /* @__PURE__ */ jsx("div", { style: { height: Math.max(0, height), ...rest == null ? void 0 : rest.style } }), DEFAULT_MARGINS$1 = [0, 0, 0, 0], getOffsetsTo = (source, target) => {
+  if(!el) return { top:0, left:0 };
   let el = source, top = -el.scrollTop, left = 0;
   for (; el && el !== target; )
     top += el.offsetTop - el.scrollTop, left += el.offsetLeft, el = el.offsetParent instanceof HTMLElement ? el.offsetParent : null;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by binoy14** [comment]
This has been fixed in [v3.46.1](https://github.com/sanity-io/sanity/releases/tag/v3.46.1)

**Comment by runeb** [comment]
Thank you for the report and the suggested patch @ahmedrowaihi! As you hopefully got notification of, we already fixed this particular issue in a recent version.
